src/components/Main.js

```diff
@@ -109,7 +119,12 @@ export default function Main({ onHandleSubmit }) {
       <Time>
         <h2>Flight Departure Date/Time</h2>
         <div>
-          <input id="flightDate" type="date" aria-label="date" />
+          <input
+            id="flightDate"
+            type="date"
+            aria-label="date"
+            min={getDate()}
```

Comment:
I can still input dates prior to today and it's accepted by the input field. In other words, it's not working 🤣

Reply:
i notice it a few minutes ago!